### PR TITLE
Relay Detail View

### PR DESCRIPTION
**Pull request by joelklabo**
Adds a new view to show relay details (NIP-11), also links to supported NIPs on GitHub

*Updated Design*

| Light      | Dark |
| ----------- | ----------- |
| light |  dark |

### CODE REVIEW

**Comment by jb55** [comment]
i missed this!

**Comment by joelklabo** [comment]
Yeah, I like it!

**Comment by jb55** [comment]
I seem to get "failed to parse nip11 data" on most relays

**Comment by joelklabo** [comment]
Hmm, I'll check it out

**Comment by joelklabo** [comment]
Ok this should be working now. I had an extra field in there that wasn't actually required. And I added a default empty value so if something is missing it still works.

For the Damus relay and the one with the yellow connection there is an actual error. Damus seems to be down and the other times out so those make sense:

Simulator Screen Recording - iPhone 14 Pro - 2023-02-04 at 14 00 13

**Comment by jb55** [comment]
will try this again

File: damus/Views/Relays/RelayDetailView.swift [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+            } else if let errorString {
+                Text(errorString)
+                    .foregroundColor(.red)
+                    .font(.caption)
+            } else {
+                ProgressView()
+            }
+        }
+        .navigationTitle(nip11?.name ?? "")
+        .onAppear {
```
Comment (jb55) [inline damus/Views/Relays/RelayDetailView.swift:62]: you can do `.task` instead of `onAppear` and then use the async version of URLSession.shared...

Comment (jb55) [reply damus/Views/Relays/RelayDetailView.swift:62]: I'll make this change don't worry about it

Comment (joelklabo) [reply damus/Views/Relays/RelayDetailView.swift:62]: Oh nice, didn't know that. Thanks!

**Pull request closed without merging**